### PR TITLE
KNJ-16282 Fix loading private key for CSOB client

### PR DESCRIPTION
**Pull request by JanaPradacova**

### CODE REVIEW

File: pycsob/client.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def __init__(self, merchant_id, base_url, private_key, csob_pub_key):
         """
         self.merchant_id = merchant_id
         self.base_url = base_url
-        self.key = self._get_key(private_key)
+        self.key = private_key
         self.pubkey = self._get_key(csob_pub_key)
```
Comment (xkubov) [inline pycsob/client.py:49]: Isn't the same problem with the public key?

Comment (rubickcz) [reply pycsob/client.py:49]: Agreed, i would do it here as well. And remove that method entirely.

Comment (JanaPradacova) [reply pycsob/client.py:49]: No it should not be a problem because we retrieve public key from file.

Comment (rubickcz) [reply pycsob/client.py:49]: I see, OK then.

**Review by rubickcz — APPROVED** [review]

**Review by Formulka — APPROVED** [review]

**Pull request merged**